improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -990,7 +986,8 @@ def _location_and_scale_parameters_to_probabilities(
 
         for index, threshold in enumerate(thresholds):
             probabilities[index, ...] = np.reshape(
-                probability_method(threshold), probabilities.shape[1:])
+                probability_method(threshold),
+                probabilities.shape[1:])  # pylint: disable=E1136
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Id 'E1136' is used to disable 'unsubscriptable-object' message emission](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4915701)